e2e/tests/uniswap_trade_test.rs

```diff
@@ -170,14 +170,14 @@ async fn test_with_ganache() {
         .call()
         .await
         .expect("Couldn't fetch TokenB's balance");
-    assert_eq!(balance, to_wei(100));
+    assert_eq!(balance, to_wei(80));
 
     let balance = token_a
         .balance_of(trader_b.address())
         .call()
         .await
         .expect("Couldn't fetch TokenA's balance");
-    assert_eq!(balance, to_wei(100));
+    assert_eq!(balance, 62500000000000000000u128.into());
```

Comment:
As a sanity check we could also check that the respective sell token has decreased by the expected amount.